esx_service/cli/vmdkops_admin_test.py

```diff
@@ -603,6 +612,11 @@ def test_tenant(self):
                                               new_name=self.tenant1_new_name)
         self.assertEqual(None, error_info)
 
+	# verify default tenant can't be renamed
+        error_info  = auth_api._tenant_update(name=DEFAULT,
+                                              new_name=self.tenant1_new_name)
+        self.assertNotEqual(None, error_info)
```

Comment:
Agree, but the error info is a string and changing the string in the code wll require changing n the test as well.